Problem1.java

```diff
@@ -0,0 +1,38 @@
+//Project Euler problem 1
+//Print the sum of all multiples of 3 and 5 beneath 1000
+
+import java.util.ArrayList;
+
+public class Problem1 {
+
+  public static void main(String[] args) {
+    int i = 1;
+    int mult = 3;
+    ArrayList<Integer> thoseMults = new ArrayList<Integer> (0);
```

Comment:
Prefer `List<Integer>` on the left-hand side. See: [Effective Java Item 52: Referr to objects by their interfaces](http://jtechies.blogspot.com/2012/07/item-52-refer-to-objects-by-their.html)